Makefile.am

```diff
@@ -175,13 +175,37 @@ endif
 jq.1: jq.1.prebuilt
 	$(AM_V_GEN) cp $(srcdir)/jq.1.prebuilt $@
 
+SUBDIRS =
+
+AM_CFLAGS += ${utf8proc_CFLAGS}
+
+if BUNDLE_UTF8PROC
+LIBJQ_SRC += modules/utf8proc/utf8proc.c
```

Comment:
`modules/utf8proc/**` should be marked as vendored in .gitattributes (ref: #2705)